cli/src/factory_ops/unboxing/config_update

```diff
@@ -224,6 +224,215 @@ function update_cluster_sls {
+    local _tgt_build=$(grep -n "target_build:"  $_release_sls_path | awk '{ print $3 }')
+    if [[ ! -z $_tgt_build ]]; then
+        line_target_build=`grep -n "target_build:"  $_release_sls_path | cut -d: -f1`
+        if [[ ! -z $line_target_build ]]; then
+            echo "Removing target_build from $_release_sls_path" >> $LOG_FILE
+            sed -ie "${line_target_build}s/.*/    target_build:/" $_release_sls_path
+            n_diff_lines=`diff $_release_sls_path ${_release_sls_path}.bkp | grep "^>" | wc -l`
+            echo "DEBUG: n_diff_lines: $n_diff_lines" >>  ${LOG_FILE}
+            if [[ $n_diff_lines != 1 ]]; then
+                echo "Could not update the target_build in $_release_sls_path, restoring the original content" | tee -a ${LOG_FILE}
+                yes | cp -f ${_release_sls_path}.bkp $_release_sls_path
+            else
+                echo "Successfully updated target_build in $_release_sls_path" >> ${LOG_FILE}
+            fi
+        else
+            echo "Could not find the target_build line number in $_release_sls_path, ignoring" >> $LOG_FILE
+        fi
+    fi
+
+    if [[ -f "${PRVSNR_ROOT}/pillar/user/groups/all/release.sls" ]]; then
+        local _release_sls_path_user=${PRVSNR_ROOT}/pillar/user/groups/all/release.sls
+        yes | cp -f $_release_sls_path_user ${_release_sls_path_user}.bkp
+        _tgt_build=$(grep -n "target_build:"  $_release_sls_path_user | awk '{ print $3 }')
+        if [[ ! -z $_tgt_build ]]; then
+            line_target_build=`grep -n "target_build:"  $_release_sls_path_user | cut -d: -f1`
+            if [[ ! -z $line_target_build ]]; then
+                echo "Removing target_build from $_release_sls_path_user" >> $LOG_FILE
+                sed -ie "${line_target_build}s/.*/    target_build:/" $_release_sls_path_user
+                # ensure the sed caused only one line change
+                n_diff_lines=`diff $_release_sls_path_user ${_release_sls_path_user}.bkp | grep "^>" | wc -l`
+                echo "DEBUG: n_diff_lines: $n_diff_lines" >>  ${LOG_FILE}
+                if [[ $n_diff_lines != 1 ]]; then
+                    echo "Could not update the target_build in $_release_sls_path_user, restoring the original content" | tee -a ${LOG_FILE}
+                    yes | cp -f ${_release_sls_path_user}.bkp $_release_sls_path_user
+                else
+                    echo "Successfully updated target_build in $_release_sls_path_user" >> ${LOG_FILE}
+                fi
+            else
+                echo "Could not find the target_build line number in $_release_sls_path_user, ignoring" >> $LOG_FILE
+            fi
+        fi
+    fi
+
+    echo "Refreshing Salt pillars" >> ${LOG_FILE}
+    salt '*' saltutil.refresh_pillar ${salt_opts}
+    echo "Checking if the target_bulid url is removed from release pillar" >> ${LOG_FILE}
+
+    _node1_tgt_build=$(salt srvnode-1 pillar.get eos_release:target_build --output=newline_values_only ${salt_opts})
```

Comment:
release pillar name needs to be changed.

Reply:
Addressed